elife/docker-scripts/docker-read-label

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 IMAGE LABEL"
+    echo "Example: $0 elifesciences/annotations_cli org.elifesciences.dependencies.api-dummy"
```

Comment:
This is a bit incomplete because we should allow to specify a tag for the `elifesciences/annotations_cli` image. Better not to depend on what `latest` is this minute.